src/System.Runtime/src/System/LazyOfTTMetadata.cs

```diff
@@ -8,7 +8,6 @@
 
 namespace System
 {
-    [Serializable]
```

Comment:
Depending on where things land with delegate, it's likely `Lazy<T>` would also need to be removed.